### PR TITLE
feat(clerk-js): Card animations

### PR DESCRIPTION
**Pull request by desiprisg**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx [hunk 1]
```diff
@@ -23,12 +23,13 @@ export const ActiveDevicesSection = () => {
   return (
     <ProfileSection.Root
       title={localizationKeys('userProfile.start.activeDevicesSection.title')}
+      centered={false}
       id='activeDevices'
     >
       <ProfileSection.ItemList id='activeDevices'>
         {!sessionsWithActivities.length && <FullHeightLoader />}
         {!!sessionsWithActivities.length &&
-          sessionsWithActivities.sort(currentSessionFirst(session!.id)).map(sa => (
```
Comment (octoper) [inline packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx:31]: This is probably remove from the linter

**Comment by desiprisg** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @desiprisg, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-m87kfja9h.clerkstage.dev) | Jan 18, 2024 02:49 PM |

**Review by JacobMGEvans — APPROVED** [review]
Formatting is also nice for the JSX 😄

**Comment by BRKalow** [comment]
@desiprisg did we decide on a way to disable the animations via the appearance prop?

**Comment by desiprisg** [comment]
> @desiprisg did we decide on a way to disable the animations via the appearance prop?

We're going to need to discuss that a bit more, unless we just add a prop for this specific effect here.

**Pull request merged**